src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/SubaccountProcessor.kt

```diff
@@ -357,14 +371,11 @@ internal open class SubaccountProcessor(
         payload: List<IndexerFillResponseObject>?,
         reset: Boolean,
     ): InternalSubaccountState {
-        val newFills = fillsProcessor.process(
+        subaccount.fills = fillsProcessor.process(
```

Comment:
[discussion] definitely not for this PR, but i imagine these Processor classes don't really need to be classes, since they're just consuming an input and returning an digested output? cc @prashanDYDX do we have plans of converting these to functions?

Reply:
IMHO, having a class instead of just static function for processor actually makes unit testing easier when there are delegations.  Prefer keeping it as-is, but we can revisit this when we have time.